datasets/guardian_authorship/guardian_authorship.py

```diff
@@ -0,0 +1,314 @@
+                           train_folder="World", valid_folder="Politics", test_folder="Society,UK"),
+
+        GuardianAuthorshipConfig(name="cross_topic_{}".format(11),
+                           version=nlp.Version("{}.0.0".format(11),
+                                               description="The Original DS with the cross-topic scenario no.{}".format(
+                                                   11)),
+                           train_folder="World", valid_folder="Society", test_folder="Politics,UK"),
+
+        GuardianAuthorshipConfig(name="cross_topic_{}".format(12),
+                           version=nlp.Version("{}.0.0".format(12),
+                                               description="The Original DS with the cross-topic scenario no.{}".format(
+                                                   12)),
+                           train_folder="World", valid_folder="UK", test_folder="Politics,Society"),
+
+
+        # # cross-genre
+        GuardianAuthorshipConfig(name="cross_genre_{}".format(1),
+                           version=nlp.Version("{}.0.0".format(13),
+                                               description="The Original DS with the cross-genre scenario no.{}".format(
+                                                   1)),
+                           train_folder="Books", valid_folder="Politics", test_folder="Society,UK,World"),
+
+        GuardianAuthorshipConfig(name="cross_genre_{}".format(2),
+                           version=nlp.Version("{}.0.0".format(14),
+                                               description="The Original DS with the cross-genre scenario no.{}".format(
+                                                   2)),
+                           train_folder="Books", valid_folder="Society", test_folder="Politics,UK,World"),
+
+        GuardianAuthorshipConfig(name="cross_genre_{}".format(3),
+                           version=nlp.Version("{}.0.0".format(15),
+                                               description="The Original DS with the cross-genre scenario no.{}".format(
+                                                   3)),
+
+                           train_folder="Books", valid_folder="UK", test_folder="Politics,Society,World"),
+
+        GuardianAuthorshipConfig(name="cross_genre_{}".format(4),
+                           version=nlp.Version("{}.0.0".format(16),
+                                               description="The Original DS with the cross-genre scenario no.{}".format(
+                                                   4)),
+                           train_folder="Books", valid_folder="World", test_folder="Politics,Society,UK"),
+    ]
+
+    # @property
+    def _info(self):
+        # TODO: Specifies the nlp.DatasetInfo object
+        return nlp.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION,
+            # nlp.features.FeatureConnectors
```

Comment:
I didn't know we had `FeatureConnectors` ;)

We have `nlp.Features` if that's what you meant

Reply:
This is from the [template for a dataset loading script](https://github.com/huggingface/nlp/blob/master/templates/new_dataset_script.py). It is commented, but that's why it appeared in my code :)